GDJS/Runtime/SpriteAnimator.ts

```diff
@@ -476,16 +476,17 @@ namespace gdjs {
       return this._animations[this._currentAnimation].name;
     }
 
-    setAnimationName(newAnimationName: string): void {
+    setAnimationName(newAnimationName: string): boolean {
```

Comment:
Add doc about the returned boolean?

Reply:
It overrides methods from the `gdjs.Animatable` interface which return `void` instead of `boolean`.
From JS events, the returned boolean can't be used. Custom objects must use `setOnFrameChangeCallback` instead.
I think it's better to keep the inherited documentation.